### PR TITLE
2.5.1 release

### PR DESCRIPTION
**Pull request by nijel**

### CODE REVIEW

**Comment by nijel** [comment]
Did I miss anything @unho ?

**Comment by nijel** [comment]
Updated to current master. I'd like to release this weekend, any feedback on this is welcome.

File: docs/conf.py [hunk 1]
```diff
@@ -65,9 +65,9 @@
 # built documents.
 #
 # The short X.Y version.
-version = '2.5.0'
+version = '2.5.1'
 # The full version, including alpha/beta/rc tags.
-release = '2.5.0'
+release = version
```
Comment (unho) [inline docs/conf.py:70]: This is not always the same.

File: docs/releases/2.5.1.rst [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+
+- Catkeys
+
+    - The catkeys format now has support for fingerprint calculation
+
+- JSON
+
+    - Allow usage of BOM in JSON files
+
+- MO
```
Comment (unho) [inline docs/releases/2.5.1.rst:47]: MO is usually including in PO, as these are Gettext.

File: docs/releases/2.5.1.rst [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+
+- PO
+
+    - Avoid adding extra space on empty comment lines
+    - Several performance improvements
+
+- Android resources
+
+    - Workaround broken plural handling for languages missing other tag
+    - Fix setting rich content with comments
```
Comment (unho) [inline docs/releases/2.5.1.rst:23]: Missing change for f2cd40b ?

File: docs/releases/2.5.1.rst [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+- TS
+
+    - Avoid duplicating plurals definition
+    - Fixed possible crash on adding new translations
+
+- INI
+
+    - Now supported on Python 3 thanks to iniparse support for it
+
+- Catkeys
```
Comment (unho) [inline docs/releases/2.5.1.rst:39]: We usually sort the formats by their importance. Catkeys is certainly not of widespread usage.

File: docs/releases/2.5.1.rst [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+
+- Android resources
+
+    - Workaround broken plural handling for languages missing other tag
+    - Fix setting rich content with comments
+
+- YAML
+
+    - Fixed handling of empty keys
+    - Improved round trip preserving of comments and style
```
Comment (unho) [inline docs/releases/2.5.1.rst:28]: Missing 2ad462e ?

**Comment by unho** [comment]
@nijel Really good. I don't think any of my comments should be a blocker, but it would be great if you can check them.

File: docs/releases/2.5.1.rst (hunk 5 above)
Comment (nijel) [reply docs/releases/2.5.1.rst:28]: This is more a bugfix for  IMHO the issue was introduced by cbf4c9c766474f9433bc988ad0818d9c522ec5ce.

File: docs/releases/2.5.1.rst (hunk 2 above)
Comment (nijel) [reply docs/releases/2.5.1.rst:47]: These are really MO specific (bbebec2608886d4e7589a94317aa371560b091e0 and 5ef74e47e900123335c2504d528e6b26774183d5)

Comment (unho) [reply docs/releases/2.5.1.rst:45]: I meant that we usually put MO related changes under PO, but stating they are MO related. I am Ok with this as is anyway.

**Pull request merged**